community/index/src/main/java/org/neo4j/index/internal/gbptree/DynamicSizeUtil.java

```diff
@@ -89,7 +93,9 @@
     private static final int FLAG_FIRST_BYTE_TOMBSTONE = 0x80;
     private static final long FLAG_READ_TOMBSTONE = 0x80000000_00000000L;
     static final int MASK_ONE_BYTE_KEY_SIZE = 0x1F;
+    static final int MASK_TWO_BYTE_KEY_SIZE = 0xFFF;
```

Comment:
according to pictures of two byte key this mask should be 1FFF, isn't it?

Reply:
The picture says e.g. `[0,1,1,k,k,k,k,k][0,k,k,k,k,k,k,k]...` where I count 12 `k`, right?